src/Microsoft.OpenApi.Hidi/readme.md

```diff
@@ -68,24 +68,28 @@ Used to convert file formats from JSON to YAML and vice versa and performs slici
 This command accepts the following parameters:
 
 	• --openapi(-d) - OpenAPI description file path in the local filesystem or a valid URL hosted on a HTTPS server 
-	• --csdl(-cs) - CSDL file path in the local filesystem or a valid URL hosted on a HTTPS server 
-	• --csdlfilter(-csf) - a filter parameter that a user can use to select a subset of a large CSDL file. They do so by providing a comma delimited list of EntitySet and Singleton names that appear in the EntityContainer. 
-	• --output(-o) - Output directory path for the transformed document 
-	• --clean-ouput(-co) - an optional param that allows a user to overwrite an existing file.  
-	• --version(-v) - OpenAPI specification version 
+	• --csdl(--cs) - CSDL file path in the local filesystem or a valid URL hosted on a HTTPS server 
+	• --csdlfilter(--csf) - a filter parameter that a user can use to select a subset of a large CSDL file. They do so by providing a comma delimited list of EntitySet and Singleton names that appear in the EntityContainer. 
+	• --output(-o) - Output directory path for the transformed document.
+    • --output-folder(--of) - The output directory path for the generated files.
+	• --clean-ouput(--co) - an optional param that allows a user to overwrite an existing file.  
+	• --version(-v) - OpenAPI specification version.
+    • --metadata-version(--mv) - Graph metadata version to use.

```

Comment:
I'm not sure we should have parameters that are Graph specific, maybe update this description?